packages/change-case/src/keys.spec.ts

```diff
@@ -63,6 +63,7 @@ const TEST_CASES: TestCase[] = [
     },
     { mergeAmbiguousCharacters: true },
   ],
+  [new Date(), 1, new Date()],
```

Comment:
Can we add tests for every condition we're checking in `isPlainObject`?

Reply:
Sure thing. Will try to find some time this week 🙂

Reply:
Added some more tests. 👍